test/ex_machina/ecto_test.exs

```diff
@@ -42,4 +42,22 @@ defmodule ExMachina.EctoTest do
       TestFactory.params_for(:user_map)
     end
   end
+
+  test "params_with_assocs/2 inserts belongs_tos that are set by the factory" do
+    assert Enum.member?(ExMachina.Article.__schema__(:associations), :editor)
```

Comment:
Reposting because the comment was buried for some reason:

> I like this, but I think extracting to a private function would be good. Something like assert has_association_in_schema?(ExMachina.Article, :editor). What do you think?

Also, I'm not exactly sure what "editor" looks like when it's not set. So it might be nice to have an assertion for that. That would also ensure that someone doesn't accidentally add it to the factory and remove test coverage